bsdlib/doc/CHANGELOG.rst

```diff
@@ -5,6 +5,20 @@ Changelog
 
 All notable changes to this project are documented in this file.
 
+bsdlib 0.6.2
+************
+
+* TLS session cache is again disabled by default due to missing support in modem firmware versions older than 1.1.1 included.
```

Comment:
Regarding first change: we can't remove "included" because neither 1.1.1 doesn't support TLS session cache. Currently there is no released modem FW out there that supports TLS session cache. @b-gent  how would you reword it then?

Reply:
Ah, sorry, I misunderstood the usage of 'included' here. 

Using 'current' might be tricky because we never know when a user is going to read it, so probably it's better to specify a version.

I suggest: "TLS session cache is now disabled by default due to missing support in modem firmware version 1.1.1 and older."

Reply:
That is ok!